config/crs/kubernetes/chains/operator_v1alpha1_chains_cr.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonChains
+metadata:
+  name: chains
+spec:
+  targetNamespace: tekton-chains
```

Comment:
Are we not useing `tekton-pipelines` ns?

Reply:
Let us use `tekton-pipelines` in our examples and keep it consistent.

technically we can put any namespace in here.

But the question we should ask is what will be the expectation of a user.
In my opinion, if a user is someone who has been using operator for some time, then they will expect to see `tekton-pipelines`. 

If someone who has been using chains and one day they decide to try operator, they might exepect `tekton-chains`.

It is more likely that majority of our users will be of first kind.

so I vote for `tekton-pipelines` 🧑‍💻

Reply:
Tested and we should be going for `tekton-pipelines` and `openshift-pipelines` namespace